src/App.js

```diff
@@ -11,14 +11,15 @@ class App extends Component {
     this.state = {
       movies: [],
       singleMovieView: null,
+      error: null,
     };
   }
 
   getAllMovies = () => {
     fetchAPI.fetchMovies()
       .then((data) => {
         this.setState({ movies: data.movies });
-      })
+      }).catch(err => this.setState({error: err}))
```

Comment:
Nice way to use the error - now that it's in state, we can easily access it to surface it to the user, for example.